app/helpers/application_helper.rb

```diff
@@ -154,4 +154,8 @@ def cache_key_heroku_slug(path)
 
     "#{path}-#{heroku_slug_commit}"
   end
+
+  def app_protocol_and_domain
```

Comment:
since this seem to never be called by itself, should it receive a `/path` and appending at the end of the string?

Reply:
Makes sense, but I think can be outside the scope of this PR.